tests/Functional/Driver/OCI8/StatementTest.php

```diff
@@ -35,24 +35,6 @@ public function testQueryConversion(string $query, array $params, array $expecte
         );
     }
 
-    /**
-     * Low-level approach to working with parameter binding
-     *
-     * @param mixed[] $params
-     * @param mixed[] $expected
-     *
-     * @dataProvider queryConversionProvider
-     */
-    public function testStatementBindParameters(string $query, array $params, array $expected): void
```

Comment:
This test was implemented in https://github.com/doctrine/dbal/pull/3738 and used to reuse the existing data provider of the `testQueryConversion()` method.

The data provider provides positional statement parameters as a list which is still supported by the "shortcut" methods like `Connection::executeQuery()` but is no longer supported by the `Statement` class being tested. At the same time, this test doesn't and didn't need to reuse the data provider which focuses on testing the SQL parser while the test itself is meant to cover binding positional parameters to the statement directly.

The test has been renamed, reworked, and moved at the end of the suite in order to not be located between the other test and its data provider.